sdcm/cluster.py

```diff
@@ -310,6 +310,16 @@ def get_startup_script(cls):
                sudo sed -i 's/#LoginGraceTime \(.*\)$/LoginGraceTime 15s/' /etc/ssh/sshd_config
                sudo systemctl restart sshd
```

Comment:
@fruch , is this line not needed now?

Reply:
I wouldn't remove it, since if you remove it, in cases that have older sshd configuration won't be used until node reboot.

better have two restarts in ubuntu 22, and not break other cases.